server/models/Profile.js

```diff
@@ -0,0 +1,46 @@
+const mongoose = require("mongoose");
+
+
+const profileSchema = new mongoose.Schema({
+ firstName: {
+   type: String,
+   trim: true
+ },
+ lastName: {
+   type: String,
+   trim: true
+ },
+ description:{
+   type: String,
+   trim:true
+ },
+ gender:{
+   type: String ,
+   trim: true
+ },
+ dateOfBirth: {
+  type: Date,
+  required: true,
+  trim: true,
+},
+phone: {
+  type: String,
+  trim: true,
+  validate: {
+    validator: function (v) {
+      return /^[0-9]{10}/.test(v);
+    },
+    message: '{VALUE} is not a valid 10 digit number!'
+  }
+},
+description:{
+  type: String,
+  trim:true
+},
+ availability:{
```

Comment:
This design for availability won't fit the use case for our project, we can revisit it at a later date though  :)

Note: We need to be able to specify availability for days of the week (Sun-Sat). Additionally, you can take time periods into consideration, Start Date - End Date (incl of start and end times)

Reply:
We can leave out availability right now 👍🏻. We'll focus more on this next week